apps/miranum-console/src/app/generate/generateAPI.ts

```diff
@@ -1,6 +1,6 @@
 import * as vscode from "vscode";
 import {getGenerateWebview} from "../../Webviews/Inputs/generateInput";
-import {generateFileMessage, generateProjectMessage} from "../../types/MessageAPI";
+import {GenerateProjectMessage} from "../../types/MessageAPI";
```

Comment:
wird generateFileMessage nicht benötigt? die untercheiden sich ja in dem Punkt type

Reply:
Der GenerateFileMessage type wurde im Code nicht verwendet, darum habe ich ihn rausgenommen.